src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -1290,7 +1290,7 @@ def compute_bootstrap_errors(
 
         if self.correct_from_presidential:
             nonreporting_units["geographic_unit_fips_p"] = nonreporting_units.geographic_unit_fips.apply(
-                lambda x: x.split("_")[1]
+                lambda x: x.split("_")[1] if "_" in x else x
```

Comment:
JFYI, this conflicts with @lennybronner 's PR 🤔  https://github.com/washingtonpost/elex-live-model/pull/143/files